.github/workflows/frontend-shared.yml

```diff
@@ -0,0 +1,44 @@
+name: frontend-shared package
+on:
+  release:
+    types:
+      - published
+jobs:
+  update-frontend-shared-version:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    # Additionally fetch master to be able to diff for changes in the frontend-shared/ folder
+    - name: Fetch origin
```

Comment:
This stumped me for a while, we can't do any `diff` without updating master. I think actions/checkout@v2 only gets the minimal git data it needs to be speedy.

Reply:
> I think actions/checkout@v2 only gets the minimal git data it needs to be speedy.

I strongly suspect you're right. We could probably optimize this fetch by only fetching changes going back for some period of time (N days?). The repository is not too large so fetching the whole thing should work to begin with.

Reply:
I noticed it only took about 1 or 2 seconds, but we would probably add a depth=100 which surely would cover to the previous tag.